public/docs/ts/latest/glossary.jade

```diff
@@ -718,33 +695,34 @@ a#snake-case
 .l-sub-section
   :marked
     An expression is a !{_Lang}-like syntax that Angular evaluates within
-    a [data binding](#data-binding).  Learn how to write template expressions
-    in the [Template Syntax](!{docsLatest}/guide/template-syntax.html#template-expressions) chapter.
+    a [data binding](#data-binding). 
+
+    Read about how to write template expressions
+    in the [Template Syntax](!{docsLatest}/guide/template-syntax.html#template-expressions) page.
 
 :marked
   ## Transpile
 .l-sub-section
   :marked
     The process of transforming code written in one form of JavaScript
-    (e.g., TypeScript) into another form of JavaScript  (e.g., [ES5](#es5)).
+    (for example, TypeScript) into another form of JavaScript  (for example, [ES5](#es5)).
```

Comment:
This is a good change. Please change "e.g." to "for example" globally. (I noticed it up above but didn't mark it...)